### PR TITLE
Remove `cgfs_types.py` for deb packages

### PR DESCRIPTION
**Pull request by libre-man**

### CODE REVIEW

File: debian/patches/fix-cgfs_types [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Description: <short summary of the patch>
```
Comment (olmokramer) [inline debian/patches/fix-cgfs_types:1]: Should we not change this and the other fields below?

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codegrade-fs",
-  "version": "1.1.2",
```
Comment (olmokramer) [inline npm-shrinkwrap.json:3]: This does not look right (and also in package.json)

File: debian/patches/fix-cgfs_types (hunk 1 above)
Comment (libre-man) [reply debian/patches/fix-cgfs_types:1]: Yeah we should, but I don't really see the use as our package misses quite a lot of information. But if you want I can add it.

Comment (olmokramer) [reply debian/patches/fix-cgfs_types:1]: Fair enough. Let's do that in another TR then and just fix it all at once

**Pull request merged**